scripts/reana_bench.py

```diff
@@ -438,7 +483,7 @@ def analyze(workflow: str, title: str) -> NoReturn:
 
 
 @cli.command()
```

Comment:
One comment to the beginning of the file. Since the file is executable, please start it with:

```script
#!/usr/bin/env python
```

Otherwise it tries to run in shell when users try `./reana_bench.py --help`...

Reply:
Done